positron/components/PositronCLH.manifest

```diff
@@ -1,3 +1,3 @@
 component {dd20d954-f2a9-11e5-aeae-782bcb9e2c3f} PositronCLH.js
 contract @mozilla.org/positron/clh;1 {dd20d954-f2a9-11e5-aeae-782bcb9e2c3f}
-category command-line-handler x-positron @mozilla.org/positron/clh;1
+category command-line-handler a-positron @mozilla.org/positron/clh;1
```

Comment:
> Sounds good. Although now that devtools is at m-devtools, I'll make this something like e-positron, so it can be overridable by something else if needed.

Erm, actually I don't need to change the entry for the Positron CLH at all anymore, since we're taking the implementation in #41. So I've left it at `m-positron`.